node/popn.go

```diff
@@ -767,6 +766,17 @@ func (nd *node) Load(ctx context.Context, args *GetArgs) (chan GetResult, error)
 				s = sel.Entries()
 			}
 
+			bal, err := nd.exch.Wallet().Balance(ctx, nd.exch.Wallet().DefaultAddress())
+			if err != nil {
+				log.Error().Err(err).Msg("error when fetching balance")
+			}
+
+			if bal.Int64() < funds.Int64() {
+				fmt.Printf("==> Fil balance: %d attofil, but offer price is %d attofil\n", bal, funds)
```

Comment:
@gallexis if the balance is too low we need to fail the transfer immediately with a proper error message. No need to log anything. Also if the balance method returns an error we also fail the transfer and return an error. Lastly, you shouldn't use int64 to compare, these might be very large numbers that may not fit so the safe way is to use the comparison method: `bal.LessThanEqual(funds)` (there will be gase costs so equal prob won't cut it either)

Reply:
Alright sounds more clear now, sorry I didn't have much intuition about how to manage things in this pr